process/machine_amd64.go

```diff
@@ -0,0 +1,10 @@
+//go:build amd64
+
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package process // import "go.opentelemetry.io/ebpf-profiler/process"
+
+import "debug/elf"
+
+const currentMachine = elf.EM_X86_64
```

Comment:
I would probably move these `currentMachine` things to `pfelf` as `CurrentMachine`. They are occasionally needed in other parts too.

Reply:
Applied. Though I could not find where to use them in other parts after a quick search.